sys/include/net/sock/dtls.h

```diff
@@ -539,6 +539,26 @@ typedef struct sock_dtls sock_dtls_t;
  */
 typedef struct sock_dtls_session sock_dtls_session_t;
 
+/**
+ * @brief   Auxiliary data provided when receiving using an DTLS sock object
+ *
+ * @details Implementations of this API may rely on this type to be compatible
```

Comment:
Isn't this more of a warning?

```suggestion
 * @warning Implementations of this API may rely on this type to be compatible
```

 Otherwise, you can just remove the `@details`. It makes it a bit more readable IMHO.

Reply:
Yes, indeed. Fixed!